### PR TITLE
lldp changes for issue#460

### PR DESCRIPTION
**Pull request by azubaidhar**
lldp changes for issue#460 ( https://github.com/openconfig/featureprofiles/issues/460 )

### CODE REVIEW

File: feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go [hunk 1]
```diff
@@ -99,14 +99,20 @@ func verifyNodeConfig(t *testing.T, nodeTelemetry *device.DevicePath, port *onda
 	if lldpEnabled != state.GetEnabled() {
 		t.Errorf("LLDP enabled got: %t, want: %t.", state.GetEnabled(), lldpEnabled)
```
Comment (sezhang2) [inline feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go:100]: Replace "LLDP enabled" with "LLDP state.GetEnabled()"

Please make similar changes to other Logf() and Error()

File: feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go [hunk 2]
```diff
@@ -99,14 +99,20 @@ func verifyNodeConfig(t *testing.T, nodeTelemetry *device.DevicePath, port *onda
...
 	}
-	if conf.GetSystemName() != state.GetSystemName() {
-		t.Errorf("LLDP SystemName got: %s, want: %s.", state.GetSystemName(), conf.GetSystemName())
+	if state.GetSystemName() != "" {
+		t.Logf("LLDP SystemName got: %s", state.GetSystemName())
+	} else {
+		t.Errorf("LLDP SystemName is not proper, got %s", state.GetSystemName())
 	}
 	if conf.GetInterface(port.Name()).GetName() != state.GetInterface(port.Name()).GetName() {
 		t.Errorf("LLDP interfaces/interface/state/name got: %s, want: %s.", state.GetInterface(port.Name()).GetName(),
```
Comment (sezhang2) [inline feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go:118]: Replace interfaces/interface/state/name with state.GetInterface(port.Name()).GetName()

File: feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go (hunk 1 above)
Comment (azubaidhar) [reply feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go:100]: lldpEnabled in if condition is a bool as TC has 2 cases one with LLDP enabled and one LLDP disabled,  code compares lldEnabled with LLDP state.GetEnabled().
 
 Are you referring if condition or message printed thru Errorf?

File: feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go (hunk 2 above)
Comment (azubaidhar) [reply feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go:118]: same as above,  state.GetInterface(port.Name()).GetName() prints actual interface name..  
do you want to replace 

" t.Errorf("LLDP interfaces/interface/state/name got: %s, want: %s.") as "t.Errorf("state.GetInterface(port.Name()).GetName() , got: %s , want: %s")

 Please let me know.

**Review by sezhang2 — APPROVED** [review]

File: feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go (hunk 2 above)
Comment (sezhang2) [reply feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go:118]: LGTM. Yes, you can update it in the next PR.

**Comment by sgirishj** [comment]
@sezhang2, could you please help merge this. Thanks!

**Pull request merged**